doc/source/ray-observability/ray-logging.rst

```diff
@@ -16,18 +48,20 @@ The log file consists of the stdout of the entrypoint command of the job.  For t
 
 .. _ray-worker-logs:
 
-Worker logs
+Worker stdout and stderr
 ~~~~~~~~~~~
-Ray's tasks or actors are executed remotely within Ray's worker processes. Ray has special support to improve the visibility of logs produced by workers.
+Ray's tasks or actors are executed remotely within Ray's worker processes. Ray has special support to improve the visibility of stdout and stderr produced by workers.
 
-- By default, all of the tasks/actors stdout and stderr are redirected to the worker log files. Check out :ref:`Logging directory structure <logging-directory-structure>` to learn how Ray's logging directory is structured.
-- By default, all of the tasks/actors stdout and stderr that is redirected to worker log files are published to the driver. Drivers display logs generated from its tasks/actors to its stdout and stderr.
+- By default, all of the tasks/actors stdout and stderr are redirected to the worker log files. This includes any log messages generated by the worker. Check out :ref:`Logging directory structure <logging-directory-structure>` to learn how Ray's logging directory is structured.
+- By default, all of the tasks/actors stdout and stderr that is redirected to worker log files is read by the driver and displayed with a call to ``print``. Drivers display all stdout/stderr generated from its tasks/actors to its own stdout/stderr.
```

Comment:
```suggestion
- By default, the driver reads the worker log files to which the stdout and stderr for all tasks and actors are redirected. To display the log files, call `print`. Drivers display all stdout and stderr generated from their tasks and actors to their own stdout and stderr.
```

Reply:
| To display the log files, call `print`

Ahh, this isn't what I meant by this passage. I meant that internally, Ray calls `print` to display stdout and stderr which are stream to the log files from the worker.

IMO this is something of an anti-pattern because it bypasses python's logging system and introduces a number of difficulties in configuring loggers. The reason I mention it here is that the fact that log file contents are printed to the stdout of the driver has implications for the user, and helps to understand the current logging system.

Reply:
Ah! I mangled that good!. I think you worded it really well in the comment, `Ray calls print to display stdout and stderr which are streamed to the log files from the worker.`